src/main/proto/umessage.proto

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright (c) 2023 General Motors GTO LLC
+ *
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+ syntax = "proto3";
+
+ package uprotocol.v1;
+ 
+ import "uri.proto";
+ import "uattributes.proto";
+ import "upayload.proto";
+ 
+ option java_package = "org.eclipse.uprotocol.v1";
+ option java_outer_classname = "UTransportProto";
+ option java_multiple_files = true;
+
+// UMessage is the envelop that wraps the topic, payload and attributes
+message UMessage {
+    UUri topic = 1;                 // The topic of the message
```

Comment:
I think we called the topic "source", it makes more sense and is easier to understand what topic actually goes here, since the sink is in the uAttributes